lesson_4/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
.vscode este un fisier de configurare a IDE-ului. Genul asta de fisiere nu le adaugam in git. Ca sa il scoti poti folosi urmatoarea comanda pe branchul acesta

`git restore --source=HEAD^ --staged  -- lesson_4/.vscode/settings.json`

comanda asta o sa mute fisierul in starea `staged` si de aici poti rula `git restore --staged` 

Gasesti mai multe detalii aici : https://devconnected.com/how-to-remove-files-from-git-commit/